packages/bierzo-wallet/src/routes/payment/components/ReceiverAddress/index.tsx

```diff
@@ -4,52 +4,38 @@ import { Theme } from '@material-ui/core';
 import Paper from '@material-ui/core/Paper';
 import TextField from '@material-ui/core/TextField';
 import { makeStyles } from '@material-ui/styles';
+import Block from 'medulas-react-components/lib/components/Block';
+import Typography from 'medulas-react-components/lib/components/Typography';
 import React from 'react';
 
 interface Props {
   readonly positionedClass?: string;
 }
 
 const useStyles = makeStyles((theme: Theme) => ({
-  paperRoot: {
+  paper: {
     display: 'flex',
     flexDirection: 'column',
     width: '100%',
-    padding: '4rem',
+    padding: '40px',
   },
 
-  toLabel: {
-    color: theme.palette.text.primary,
-    fontFamily: '"Muli SemiBold", sans-serif',
-    fontSize: '1.4rem',
-  },
-
-  textFieldRoot: {
+  textField: {
     width: '100%',
-    height: '5rem',
-    margin: '1.6rem 0 .8rem 0',
-  },
-
-  validityLabel: {
-    color: theme.palette.error.main,
-    fontFamily: '"Muli SemiBold", sans-serif',
-    fontSize: '1.4rem',
+    height: '50px',
+    margin: '16px 0 8px 0',
   },
 
   tooltipContainer: {
     display: 'flex',
     alignSelf: 'flex-end',
-    fontSize: '1.4rem',
-    marginTop: '2.4rem',
-  },
-
-  tooltipLabel: {
-    color: theme.palette.text.primary,
+    fontSize: '14px',
```

Comment:
What about remove the fontSize here and include it in the FontAwesomeIcon (if needed) and also in the Typography component using the variant prop.

Do the same with the marginTop prop, is better to wrap it using a block and specify the margin via props.